examples/eia/docs/data/country-interchange.csv.js

```diff
@@ -1,30 +1,27 @@
+import "dotenv/config";
 import * as d3 from "d3";
 
+// Access key from .env
+const {EIA_KEY} = process.env;
+
 const end = d3.timeDay.offset(d3.timeHour(new Date()), 1);
 const start = d3.timeHour(d3.utcDay.offset(end, -7));
-const convertDate = d3.timeFormat("%m%d%Y %H:%M:%S"); // Convert dates into the format the EIA web API expects
+const convertDate = d3.timeFormat("%Y-%m-%dT%H"); // Convert dates into the format the EIA web API expects
```

Comment:
I expect these should all be using `d3.utc` instead of `d3.time`? I don’t think it should matter much and it’s a little confusing since in CI we hard-code the America/Los_Angeles time zone. Anyway, fine to leave it as-is, too.